acquia_lift.services.yml

```diff
@@ -11,6 +11,9 @@ services:
   acquia_lift.service.helper.help_message_helper:
     class: Drupal\acquia_lift\Service\Helper\HelpMessageHelper
     arguments: ['@config.factory', '@link_generator']
+  acquia_lift.service.helper.lift_api_helper:
+    class: Drupal\acquia_lift\Service\Helper\LiftAPIHelper
+    arguments: ['@config.factory']
```

Comment:
Well, I don't see the lift-sdk-php to be only Decision API, so I rather call it api/LiftApi perhaps